### PR TITLE
Add Dreamshaper to manifest, fix manifest scripts

### PR DESCRIPTION
**Pull request by ArendJanKramer**
Update create_manifest.dart to recursively scan for filesizes in public models, and to use api instead of html parser. 
Flip default RGB/BGR for TTI

Unfortunately, there's no collection yet for image generation by openvino. I made my own, and a small adjustment to make the code work with foreign collections.

### CODE REVIEW

File: assets/manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     {
       "name": "Mistral 7b Instruct V0.1",
       "id": "mistral-7b-instruct-v0.1-int8-ov",
-      "fileSize": 7824223238,
+      "fileSize": 7287479393,
       "optimizationPrecision": "int8",
       "contextWindow": 32768,
       "description": "Chat with Mistral 7b Instruct V0.1",
```
Comment (Radwan-Ibrahim) [inline assets/manifest.json:9]:
```suggestion
      "description": "Unlock the Power of AI on Your PC: Start Chatting with the Mistral 7b Instruct",
```

File: assets/manifest.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     {
       "name": "Whisper Base",
       "id": "whisper-base-fp16-ov",
-      "fileSize": 263786768,
+      "fileSize": 251616604,
       "optimizationPrecision": "fp16",
       "contextWindow": 0,
       "description": "Transcribe video with Whisper Base",
```
Comment (Radwan-Ibrahim) [inline assets/manifest.json:31]:
```suggestion
      "description": "We make it easy to connect with people: Use Whisper to transcribe videos.",
```

File: assets/manifest.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
     {
       "name": "Phi 3 Mini 4k Instruct",
       "id": "Phi-3-mini-4k-instruct-int4-ov",
-      "fileSize": 2317366276,
+      "fileSize": 2159924401,
       "optimizationPrecision": "int4",
       "contextWindow": 4096,
       "description": "Chat with Phi 3 Mini 4k Instruct",
```
Comment (Radwan-Ibrahim) [inline assets/manifest.json:20]:
```suggestion
      "description": "Unlock AI Power: Experience the Phi 3 Mini Model on Your PC",
```

File: lib/public_models.dart [hunk 4]
```diff
@@ -34,10 +34,17 @@ void writeProjectJson(PublicProject project) {
...
   final configJsonURL = huggingFaceModelFileUrl(project.modelId, "config.json");
-  final info = (await http.get(Uri.parse(configJsonURL)));
-  if (info.statusCode == 200){
-    final config = jsonDecode(info.body);
+  final response = await http.get(
+      Uri.parse(configJsonURL),
+      headers: {
+        "Authorization":"Bearer ${project.modelInfo!.collection.token}",
+      }
+  );
```
Comment (RHeckerIntel) [inline lib/public_models.dart:42]: No need for collection token anymore right?

File: assets/manifest.json [hunk 5]
```diff
@@ -34,33 +34,231 @@
...
-      "description": "Generate images with Dreamshaper V7",
-      "task": "text-to-image",
+      "description": "Transcribe video with Distil Whisper Large V2",
+      "task": "speech",
       "author": "OpenVINO",
-      "collection": ""
+      "collection": "speech-to-text-672321d5c070537a178a8aeb"
+    },
+    {
+      "name": "Distil Whisper Large V2",
```
Comment (RHeckerIntel) [inline assets/manifest.json:72]: Distil whisper models do not work with genai. They throw  error:
```
FrontEnd API failed with GeneralFailure:
ir: Could not open the file: "/Users/rhecker/Library/Containers/com.openvino.console/Data/Library/Application Support/com.openvino.console/454677ae-b353-47d9-bb8b-d602e60aab32/openvino_decoder_with_past_model.xml"
```

File: scripts/create_manifest.dart [hunk 6]
```diff
@@ -174,10 +198,12 @@ void generate() async {
...
     "whisper-base-fp16-ov",
     "open_llama_3b_v2-int8-ov",
+    "LCM_Dreamshaper_v7-int8-ov",
   ];
   final List<Collection> collections = [
-    Collection("speech-to-text-672321d5c070537a178a8aeb", "OpenVINO", "speech", (String name) => "Transcribe video with $name"),
-    Collection("llm-6687aaa2abca3bbcec71a9bd", "OpenVINO", "text-generation", (String name) => "Chat with $name"),
+    Collection("speech-to-text-672321d5c070537a178a8aeb", "OpenVINO", "OpenVINO", "speech", (String name) => "Transcribe video with $name"),
+    Collection("llm-6687aaa2abca3bbcec71a9bd", "OpenVINO", "OpenVINO", "text-generation", (String name) => "Chat with $name"),
+    Collection("image-generation-6763eab8ac097237330c78c5", "arendjan", "OpenVINO", "text-to-image", (String name) => "Generate images with $name"),
```
Comment (RHeckerIntel) [inline scripts/create_manifest.dart:206]: Is it okay to publish your Collection like this? Please check.

File: assets/manifest.json [hunk 7]
```diff
@@ -34,33 +34,231 @@
...
+      "collection": "llm-6687aaa2abca3bbcec71a9bd"
+    },
+    {
+      "name": "LCM_Dreamshaper_v7",
+      "id": "LCM_Dreamshaper_v7-int8-ov",
+      "fileSize": 1076861241,
+      "optimizationPrecision": "int8",
       "contextWindow": 0,
-      "description": "Generate images with Dreamshaper V7",
+      "description": "Generate images with LCM_Dreamshaper_v7",
```
Comment (Radwan-Ibrahim) [inline assets/manifest.json:53]:
```suggestion
      "description": "Boost Your Creativity: Generate Images with Dreamshaper on Your PC",
```

**Review by Radwan-Ibrahim — COMMENTED** [review]
A few minor comments

File: scripts/create_manifest.dart [hunk 8]
```diff
@@ -89,50 +109,54 @@ class ModelInfo {
...
-        continue;
-      }
-      //print(text);
-      final match = pattern.firstMatch(text);
-      if (match == null){
-        continue;
-      }
+  static Future<Map<String, int>> getFileSizesRecursively(
+      String id, String author, String path) async {
+    String url = "https://huggingface.co/api/models/$author/$id/tree/main";
```
Comment (RHeckerIntel) [inline scripts/create_manifest.dart:119]: Damn. I missed that api 🤣  

I do appreciate that I'm reviewing this at a garage, whilst writing that original code in a garage too.

**Review by RHeckerIntel — APPROVED** [review]

File: assets/manifest.json (hunk 1 above)
Comment (ArendJanKramer) [reply assets/manifest.json:9]: Auto generated, sorry

File: assets/manifest.json (hunk 2 above)
Comment (ArendJanKramer) [reply assets/manifest.json:31]: Auto generated, sorry

File: assets/manifest.json (hunk 3 above)
Comment (ArendJanKramer) [reply assets/manifest.json:20]: Auto generated, sorry

File: assets/manifest.json (hunk 5 above)
Comment (ArendJanKramer) [reply assets/manifest.json:72]: How to fix, as they're in the official collection?

File: scripts/create_manifest.dart (hunk 6 above)
Comment (ArendJanKramer) [reply scripts/create_manifest.dart:206]: I dunno

**Pull request merged**